operator/pkg/model/translation/ingress/dedicated_ingress.go

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
I think the commit message should be "The core logic is same as **shared** translator, except:"